aspnetcore/fundamentals/logging/index/samples/2.x/TodoApiSample/Startup.cs

```diff
@@ -28,18 +33,30 @@ public void ConfigureServices(IServiceCollection services)
             services.AddMvc();
 
             // Add our repository type
-            services.AddScoped<ITodoRepository, TodoRepository>();
+            services.AddSingleton<ITodoRepository, TodoRepository>();
+            _logger.LogInformation("Added TodoRepository to services");
         }
 
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
```

Comment:
This comment exceeds the 85 character limit for a line of code. Consider removing it.